### DISCUSSION TITLE
posts/odoo/Odoo-xml-invisible

### DISCUSSION THREAD

**Discussion by giscus** [dreamhunter2333/dreamhunter-blog#23] [General]
# posts/odoo/Odoo-xml-invisible

你指尖跃动的电光，是我此生不变的信仰

https://dreamhunter2333.com/posts/odoo/Odoo-xml-invisible.html

**Comment by dreamhunter2333** [comment]
> 迁移数据
> "createdAt": "2023-08-15T10:50:36.116Z",
> "nick": "Anonymous",
> "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36"

`attrs=\"{'invisible': [('is_who', '=', 2)]}`
